content/docs/user-guide/components/reference/physx/static-rigid-body.md

```diff
@@ -0,0 +1,17 @@
+---
+linkTitle: PhysX Static Rigid Body
+title: PhysX Static Rigid Body Component
+description: The PhysX Static Rigid Body component creates a static solid object that cannot move and collides with other PhysX entities.
+toc: true
+---
+
+The **PhysX Static Rigid Body** component makes an entity a static solid object that cannot move and collide with other PhysX entities. The entity must also have at least one [PhysX Shape Collider](/docs/user-guide/components/reference/physx/shape-collider/)  or [PhysX Collider](/docs/user-guide/components/reference/physx/collider/) component that defines a collider for the entity.
```

Comment:
nit: The way this is written it reads a little like `'cannot move and collide'` (so seems like it can't move and also cannot collide with anything). Is that correct? I thought it could collide with simulated rigid bodies? As it needs a collider itself. Consider rewriting to `'cannot move. It can collide with other....'` so it breaks the two parts up (if that makes sense).

Reply:
Great feedback, I went with the same sentence as in Dynamic Rigid Body page, but in there both and "can do" so there is no confusion there :) I'll use your suggestion in this page. Thanks.